modules/040-node-manager/hooks/generate_webhook_certs.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2023 Flant JSC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hooks
+
+import (
+	"github.com/deckhouse/deckhouse/go_lib/hooks/tls_certificate"
+)
+
+const (
+	cn = "capi-controller-manager-webhook"
+)
+
+var _ = tls_certificate.RegisterInternalTLSHook(tls_certificate.GenSelfSignedTLSHookConf{
+	SANs: tls_certificate.DefaultSANs([]string{
+		"capi-webhook-service.d8-cloud-instance-manager.svc",
+		"capi-webhook-service.d8-cloud-instance-manager.svc.cluster.local",
```

Comment:
We have helper for cluster domain SAN's
https://github.com/deckhouse/deckhouse/blob/main/go_lib/hooks/tls_certificate/cert_helpers.go#L35

Using as [here](https://github.com/deckhouse/deckhouse/blob/e834bcd452f75193f0228a97b11b6c398e745398/modules/150-user-authn/hooks/order_dex_tls_certificate.go#L29)

Reply:
Also we add `capi-webhook-service.d8-cloud-instance-manager` without svc. See here https://github.com/deckhouse/deckhouse/blob/e834bcd452f75193f0228a97b11b6c398e745398/modules/150-user-authn/hooks/order_dex_tls_certificate.go#L27

But probably it is unnecessary.

Reply:
added